### PR TITLE
PR for version 1.1

### PR DESCRIPTION
**Pull request by maxibor**
### Changes

- Update Sourcepredict to version 0.4 and reflect new parameters in coproID (version updated in [#19](https://github.com/nf-core/coproid/pull/19)), implemented with e4afca7 c2d4164 

- Changed bedtools bamtofastq to samtools fastq e4afca7

- Fixed column names in report (PC* to DIM* ) e4afca7

- Update README to inlude Zenodo badge (for #18 ), Quick start, contributor section, and tools references. 9874ae8 e85988b

- Update documentation bedfdde

- Update Nextflow minimum version to 19.04.0 44999fd 62cb55d 
- Update travis for more recent nextflow requirements 1e3e454

## PR checklist

 - [X] This comment contains a description of changes (with reason)
 - [X] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [X] Make sure your code lints (`nf-core lint .`).
Linting failing because of Docker *dev* image tag
```
ERROR: Test Failures:
  http://nf-co.re/errors#5: Continuous integration must check for master/patch branch PRs: './.travis.yml'
```
 - [X] Documentation in `docs` is updated
 - [X] `CHANGELOG.md` is updated
 - [X] `README.md` is updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,49 @@
...
+[![Documentation Status](https://readthedocs.org/projects/coproid/badge/?version=latest)](https://coproid.readthedocs.io/en/latest/?badge=latest) 
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2653756.svg)](https://doi.org/10.5281/zenodo.2653756)
+
+
 
 ![logo_nf_core](assets/img/coproID_nf-core_logo.svg)
 
+## Introduction
+
+**CoproID** helps you to identify the _"true maker"_ of a sequenced Coprolite by checking the microbiome composition and the endogenous DNA.
```
Comment (jfy133) [inline README.md]: Maybe specify NGS, or Illumina sequenced Coprolite. A lot of people still run PCR data

File: main.nf [hunk 2]
```diff
@@ -254,11 +256,19 @@ if (params.library == 'classic'){
     library = '--UDGhalf'
 }
 
-if( ! nextflow.version.matches(">= 0.30") ){
+if( ! nextflow.version.matches(">= 19.04.0") ){
     println "Your version of Nextflow is too old, please update to Nextflow >= 0.30"
```
Comment (jfy133) [inline main.nf]: Your error statement is wrong, even though condition is correct.

File: main.nf [hunk 3]
```diff
@@ -254,11 +256,19 @@ if (params.library == 'classic'){
...
     exit(1)
 }
 
+// Check sourcepredict parameters
+
+if (params.sp_embed != 'mds' && params.sp_embed != 'tsne' && params.sp_embed != 'umap'){
+    println "${params.sp_embed} is not a valid method for SourcePredict embedding"
+    println "Available methods are: mds, tsne, umap"
```
Comment (jfy133) [inline main.nf]: Small suggestion: as nextflow best practise is to put the strings in quotes on the command itself, maybe put the quotes around each moethod in line 268

File: main.nf [hunk 4]
```diff
@@ -616,12 +626,12 @@ process bam2fq {
             out1 = name+"_"+params.name1+".unaligned_R1.fastq"
             out2 = name+"_"+params.name1+".unaligned_R2.fastq"
             """
-            bedtools bamtofastq -i $bam -fq $out1 -fq2 $out2
+            samtools fastq -1 $out1 -2 $out2 $bam
```
Comment (jfy133) [inline main.nf]: I think you might be missing a re-direct here: `samtools fastq 1.fq 2.fq > $bam

File: main.nf (hunk 2 above)
Comment (maxibor) [reply main.nf]: Updated to use the version in manifest b95bc5e 
```
if( ! nextflow.version.matches(workflow.manifest.nextflowVersion) ){
    println "Your version of Nextflow is too old, please use Nextflow "+workflow.manifest.nextflowVersion.toString()
    exit(1)
}
```

File: main.nf (hunk 3 above)
Comment (maxibor) [reply main.nf]: Done 8d97a46

File: README.md (hunk 1 above)
Comment (maxibor) [reply README.md]: Done b57b290

File: main.nf (hunk 4 above)
Comment (maxibor) [reply main.nf]: I made it now more verbose with c2d4164 , but this is the correct syntax.

Example from the [samtools doc](http://www.htslib.org/doc/samtools.html):
`samtools fastq -1 paired1.fq -2 paired2.fq -0 /dev/null -s /dev/null -n -F 0x900 in.bam`

Comment (jfy133) [reply main.nf]: Oh interesting. I assumed you had missed it because the command below does use `>`, but good to know.

**Pull request merged**